sample/Droidcon17/speakers

```diff
@@ -0,0 +1,989 @@
+[{
+      "id": 1,
+      "name": "Giorgio Natili",
+      "email": "",
+      "mobile": "",
+      "photo": "/images/speakers/gn.jpg",
+      "sessions": [{
+         "id": 1,
+         "title": "Welcome: Community Addiction"
+      }],
+      "organisation": "Akamai Technologies",
+      "country": "",
+      "short_biography": "On paper, Giorgio Natili is an engineering lead at Akamai Technologies where he spearheads the implementation of the new web apps for the security products line of Akamai and leads the development of a mobile SDK. On the job, Giorgio is a strong proponent of agile development practices whose passion for usable, maintainable and testable code is only surpassed by his determination to make things work. His previous speaking engagements include Adobe Max, 360|Flex, FITC, Mobile Web Dev Conference, Mobile Tea, Agile Alliance Conference, Codemotion, etc.",
+      "website": "",
+      "twitter": "",
+      "facebook": "",
+      "github": "",
+      "linkedin": "",
+      "featured": false,
+      "position": "",
+      "city": "",
+      "gender": "",
```

Comment:
@heysadboy I used scripts to scrape the information. Let me see how I can handle this in a better way to incorporate custom information for different speakers..